src/features/client/components/Header/index.jsx

```diff
@@ -54,44 +59,107 @@ function Header() {
           justifyContent="space-between"
           alignItems="center"
         >
-          <Flex flexWrap="wrap" gap="10px" alignItems="center">
+          <Flex flexWrap="nowrap" gap="10px" alignItems="center">
```

Comment:
instead of having 2 places with conditional rendering for "delivery to" you could leave "wrap" here and set width for container at which it wraps, less boilerplate, easier to maintain.